prow/plugins.yaml

```diff
@@ -206,6 +207,9 @@ plugins:
 
   kubernetes/dns:
   - trigger
+  
+  kubernetes/examples:
+  - trigger
```

Comment:
we might need `approve` here too. I've been trying to get OWNERS files for each sample.

Reply:
@ahmetb fixed https://github.com/kubernetes/test-infra/pull/7455/commits/8f9ac437ef7d5dbc7b7e3cdc7ac0b272e6b91924

Reply:
I think you only need `trigger` if you are running tests on prow, which doesn't seem to be the case currently.